### ISSUE TITLE
Pivot overflow aria-label is not being passed down to the overflow button

### ISSUE THREAD

**Issue by linazar** [microsoft/fluentui#19835]
### Environment Information

- **Package version(s)**: Fluent React 8.34.2
- **Browser and OS versions**: Edge 93.0.961.47 ; Windows 10 Enterprise

### Describe the issue:

.ms-Pivot-overflowMenuButton does not inherit aria-label from Pivot overflow aria-label

### Please provide a reproduction of the issue in a codepen:

Run Accessibility FastPass on https://developer.microsoft.com/en-us/fluentui#/controls/web/pivot 

#### Actual behavior:

overflowMenuButton does not have discernible text

#### Expected behavior:

overflowMenuButton should inherit aria-label from Pivot

### Documentation describing expected behavior

Accessibility Insights for Web

**Comment by khmakoto** [comment]
Hi @linazar, I don't see this issue when running the Fast Pass of Accessibility Insights for Web on the website as you suggested. Can you confirm if this is still something you're experiencing?

Below is a screenshot of what I see, which are errors related to the website and not to the actual component:
![image](https://user-images.githubusercontent.com/7798177/142092642-a0ac6cb5-291f-4af4-a1e6-ebb331bfceda.png)

**Comment by linazar** [comment]
Thank you @khmakoto ! Yes, it looks like this is fixed in the newest Fluent UI version, we no longer have this issue. Thanks so much!

**Comment by khmakoto** [comment]
Sounds good @linazar! I'll close the issue then. Let us know if there is anything else we can do!

**Comment by linazar** [comment]
@khmakoto There actually are three more accessibility issues with the overflow component - should I create a separate bug for them? They reproduce in the most current fluent ui (8.42.3). If you go to https://developer.microsoft.com/en-us/fluentui#/controls/web/pivot and scroll all the way down to "Pivot with overflow menu", click on the three dots so that the "The Last Tab" is visible, while it's visible - run FastPass. It gives three issues: aria-required-children, aria-required-parent
, and  duplicate-id-active.

**Comment by khmakoto** [comment]
@linazar if you could open separate issues for those that would be awesome!
